src/ppscore/calculation.py

```diff
@@ -158,7 +185,7 @@ def _infer_task(df, x, y):
     if category_count == 2:
         return "classification"
     if category_count == len(df[y]) and (
-        is_string_dtype(df[y]) or is_categorical_dtype(df[y])
```

Comment:
We are using black as a default formatter

Reply:
Will take care of it :)

Reply:
Great, thank you. We also have to document this somewhere

Reply:
you do not seem to use any specific settings for black, right ? (nothing I found in the repo at least)

Reply:
Exactly, just the standard